src/meltano/core/bundle/settings.yml

```diff
@@ -34,6 +34,9 @@ settings:
   kind: integer
   value: 10485760 # 10 MiB
 - name: python
+- name: auto_install
+  kind: boolean
+  value: true
```

Comment:
I think this needs an update to the JSON schema file

Reply:
https://github.com/meltano/meltano/pull/8482/commits/510b9fea7198d61f045598650a99f790203ca88a